boards/mega-xplained/doc.txt

```diff
@@ -10,27 +10,55 @@ is an evaluation kit by Atmel (now Microchip) for their ATmega1284P microcontrol
 
 ### Flash the board
 
-1. The board may be flashed through JTAG or using a SPI ISP programmer. If
-   the Buspirate is being used, then `make flash` can be used to flash the
-   board:
-```
-     make BOARD=mega-xplained -C examples/hello-world flash
-```
+You can change the fuses and flash the board using an SPI ISP programmer such
+as the Bus Pirate, or using a JTAG programmer such as the Atmel-ICE with a
+JTAG adapter.
+
+If using the Atmel-ICE as a JTAG programmer, connect the adapter as shown in
+the image below.
+
+   [photo](https://photo.jpg)
+
+1. First, make sure the default fuse settings are correct.<br/>
+
+   WARNING: setting the fuses incorrectly can brick your board!
 
-2. The default fuse settings must also be changed.<br/>
    If using the Buspirate:
+
 ```
      avrdude -p m1284p -c buspirate -P /dev/ttyUSB0 -U efuse:w:0xFF:m
      avrdude -p m1284p -c buspirate -P /dev/ttyUSB0 -U hfuse:w:0x99:m
      avrdude -p m1284p -c buspirate -P /dev/ttyUSB0 -U lfuse:w:0xE2:m
 ```
-WARNING: setting the fuses incorrectly can brick your board!
+
+   If using the Atmel-ICE:
+
+```
+     avrdude -p m1284p -c atmelice -U efuse:w:0xFF:m
+     avrdude -p m1284p -c atmelice -U lfuse:w:0x99:m
+     avrdude -p m1284p -c atmelice -U lfuse:w:0xE2:m
```

Comment:
I have the impression there's a typo here (two times `lfuse`), if one does that the board might be bricked.

Reply:
thanks! good catch for sure.

Reply:
That what "testing procedure" is for :) To run the procedure on your machine and print the output before review ;)

Reply:
Thankfully he didn't though!